_data/simple-icons.json

```diff
@@ -15170,6 +15170,16 @@
             "hex": "FF0000",
             "source": "https://partnermarketinghub.withgoogle.com/#/brands"
         },
+        {
+            "title": "YouTube Shorts",
+            "hex": "FF0000",
+            "source": "https://www.youtube.com/shorts",
+            "aliases": {
+                "aka": [
```

Comment:
@adamrusted
Do we require this aka for Youtube Shorts?

Reply:
Nope. Because the word 'Shorts' is in the title - it'll already be found without the alias.